flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/functions/scalar/ToTimestampLtzFunction.java

```diff
@@ -92,74 +121,72 @@ public TimestampData eval(Integer epoch) {
             return null;
         }
 
-        return eval(epoch, 3);
```

Comment:
Replying to this comment:
This is not possible, because we handle exact numeric and fractional data differently. Therefore, we can’t use the umbrella Number for them. 
I keep them as is to be consistent with existing logic. 
@snuyanzin , if you have a better way, please shed some lights here. 


> Can not we replace a number of these duplicating methods with something like
> 
>     public TimestampData eval(Number epoch) {
>         if (epoch == null) {
>             return null;
>         }
> 
>         return eval(epoch, 3);
>     }

Reply:
Resolved.